packages/shared_preferences/shared_preferences_platform_interface/lib/shared_preferences_platform_interface.dart

```diff
@@ -48,6 +48,11 @@ abstract class SharedPreferencesStorePlatform extends PlatformInterface {
   @Deprecated('Use MockPlatformInterfaceMixin instead')
   bool get isMock => false;
 
+  /// Sets prefix for necessary platforms (web).
```

Comment:
Do we really need this for web? Couldn't we just eliminate `_checkPrefix`? The idea of the assert doesn't seem all that useful in a world where the prefix can be changed (in particular where it can be changed to `''`).

Reply:
That was my initial plan for web. I'd certainly prefer to avoid having any mention of prefix anywhere outside the main package.